deployment/kubernetes-play-scala-grpc-example.yml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: "apps/v1beta2"
+kind: Deployment
+metadata:
+  name: "play-scala-grpc-example-v1-0-snapshot"
+  labels:
+    appName: "play-scala-grpc-example"
+    appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+  template:
+    metadata:
+      labels:
+        appName: "play-scala-grpc-example"
+        appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: "play-scala-grpc-example"
+          image: "play-scala-grpc-example:1.0-SNAPSHOT"
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: "JAVA_OPTS"
+              value: "-Dplay.http.secret.key=a-very-strong-key-for-production -Dplay.filters.hosts.allowed.0=myservice.example.org -Dplay.filters.hosts.allowed.1=192.168.99.100"
```

Comment:
Maybe here we could add a comment saying that the secret.key must be changed and unique for each application.

Reply:
See #41 . Once this and #40 are merged we'll extract the settings override to a `prod-application.conf` which will make documenting production values a lot easier and keep the yaml files easier to read.